### PR TITLE
Modify DfBuilder to include "Numeric" label types

### PR DESCRIPTION
**Pull request by gorcha**
To resolve issue #26 - added support for "Numeric" label types.

### CODE REVIEW

**Comment by hadley** [comment]
Thanks! Would you mind also adding a test?

**Comment by gorcha** [comment]
No worries, shall do!

Danny

**Comment by gorcha** [comment]
Just confirming tests have been added in the last commit.

File: tests/testthat/test-read-sas.R [hunk 1]
```diff
@@ -14,6 +14,13 @@ test_that("value labels parsed from bcat file", {
   expect_equal(attr(df$workshop, "labels"), c(R = 1, SAS = 2))
 })
 
+test_that("value labels read in as same type as vector", {
+  df <- read_sas("hadley.sas7bdat", "formats.sas7bcat")
+
+  expect_equal(class(as.vector(df$gender)), class(attr(df$gender, "labels")))
```
Comment (hadley) [inline tests/testthat/test-read-sas.R]: It'd be better to use `typeof()` here instead of `class(as.vector(x))`.  But I can do that

Comment (hadley) [inline tests/testthat/test-read-sas.R]: It would be better to use `typeof()` here (and below), instead of `class(as.vector(x))`

**Comment by hadley** [comment]
Could you please make that one last change and add a bullet point to NEWS? Thanks!

**Comment by gorcha** [comment]
Sure thing. Would you like a version number change or would you like it under 0.1.1.9000?

**Comment by hadley** [comment]
Keep it under 0.1.1.9000 - I don't usually bother bumping dev versions unless another dev package needs to depend on the update.

**Comment by gorcha** [comment]
No worries, done! Thanks for the advice.

**Comment by hadley** [comment]
Thanks!

**Comment by magicalisabel** [comment]
Hi! I'm new to R and have no experience in computer programming. I had a similar problem to Hadley's #26 . It happened after I read_dta using Haven function. Then I reshaped the data to wide format, and use View function, and the error message occurred. I cannot understand the pull you wrote here because I really no nothing about computer programming! Could you help me figure out how to solve my problem? Also, weirdly, this problem only occurred on one of my computers. Both computers have windows system and the same version of R studio.

**Pull request merged**